src/js/modules/utils.js

```diff
@@ -71,11 +71,24 @@ function sortObject (arr) {
  * @param  {object} embeds Sorted array of iframe html
  * @return {string}        String to be rendered
  */
-export function createText (str, embeds) {
-  let sortedEmbeds = sortObject(embeds)
-  for (let i = 0; i < sortedEmbeds.length; i++) {
-    str += ` ${sortedEmbeds[i].text}`
-  }
+export function createText (str, embeds) 
+{
+	let sortedEmbeds = sortObject(embeds)
+	if(str=='' && sortedEmbeds.length>0)
+	{
+		str += `${sortedEmbeds[0].text}`
+		for(let i = 1; i < sortedEmbeds.length; i++) 
+		{
+			str += ` ${sortedEmbeds[i].text}`
+		}
+	}
+	else
+	{
+  		for (let i = 0; i < sortedEmbeds.length; i++) 
+  		{
+    			str += ` ${sortedEmbeds[i].text}`
+  		}
+	}
```

Comment:
Try to simplify this. Instead of whole if and else

```js
for (let i = 0; i < sortedEmbeds.length; i++) {
  const x = sortedEmbeds[0].text
  str += (!str && i===0) ?  `${x}` : ` ${x}`
}
```